node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
+    /**
+     * Fully-qualified name of the translator (required).
+     */
+    public readonly name: string;
+
+    /**
+     * List of interfaces implemented by the translator.
+     */
+    public readonly interfaceNames: string[];
+
+    /**
+     * Name of the onboarding module required by the translator.
+     */
+    public readonly onboardingName: string;
```

Comment:
I wasn't sure "module" is the correct terminology, since that implies certain things in the node.js context.

I also considered `onboarderName`, since "onboarder" seems like a natural parallel to "translator", but it sounds funny.

---
In reply to: [71904172](https://github.com/opent2t/opent2t/pull/1#discussion_r71904172) [](ancestors = 71904172)